src/java/fr/paris/lutece/portal/web/download/DownloadServlet.java

```diff
@@ -42,7 +44,11 @@ protected void doGet( HttpServletRequest request, HttpServletResponse response )
         {
             response.sendRedirect( AppPathService.getSiteMessageUrl( request ) );
         }
-        
+        catch( UserNotSignedException e )
+        {
+            response.sendRedirect( PortalJspBean.redirectLogin( request ) );
```

Comment:
This should probably use response.encodeRedirectURL on the redirectLogin result

Reply:
You must call sendRedirect with the result of encodeRedirectURL. See https://github.com/lutece-platform/lutece-auth-plugin-mylutece/blob/b7d18baba4784157c0131d4b4972450936ea2592/webapp/jsp/site/plugins/mylutece/DoMyLuteceLogout.jsp#L5 for instance